README.md

```diff
@@ -23,6 +23,11 @@ Do not create issues on GitHub if you need help. Issues are for bug reporting an
 <img src="https://github.com/Project-Sloth/ps-dispatch/assets/82112471/01569df8-d5f6-417b-bcd4-422551eaa840" width="600">
 <img src="https://github.com/Project-Sloth/ps-dispatch/assets/82112471/f2b111b2-60c3-428e-b12a-1bfed617f09e" width="800">
 
+# Change Language.
+
+- Place this `setr ox:locale en` inside your `server.cfg`.
+- Change the `en` to your desired language! `en = English, de = German`
+
```

Comment:
Could you also add a table with the alias of the language plus the language name? Will make it easy to figure out what languages are supported.